docs/quick-start.rst

```diff
@@ -37,7 +37,35 @@ Select ``Add targets`` from the top navigation bar. Artemis takes input in the f
 of entries separated with newlines. Artemis works with both IPs and domains. If
 a URL is provided, the domain from this URL will be scanned.
 
+To be later able to filter various types of targets, provide a tag in the `Tag` field.
+
 Viewing results
 ---------------
 
 To view results, click the ``View results`` link in the top navigation bar.
+
+.. _generating-e-mails:
+
+Generating e-mails to be sent
+-----------------------------
+Artemis has a feature to generate e-mail reports containing a description of found vulnerabilities.
+
+These reports won't contain everything found by Artemis - custom logic (residing in
+``artemis/reporting/modules/``) will make an educated guess whether a vulnerability
+is a true positive and interesting enough to be reported.
+
+To generate these e-mails, use:
+
+``./scripts/export_emails ALREADY_EXISTING_REPORT_DIRECTORY TAG LANGUAGE``
+
+ - ``ALREADY_EXISTING_REPORT_DIRECTORY`` is a directory where JSON files produced by previous script invocations
+   reside. This allows you to skip sending messages that have already been sent.
```

Comment:
Just `REPORT_DIRECTORY`.

Reply:
Can I do PREVIOUS_REPORTS_DIRECTORY ? it's shorter and I don't want it to be confused with some kind of output directory.